docs/dg/workflow.md

```diff
@@ -99,9 +99,11 @@ See [vue-devtools project home](https://github.com/vuejs/vue-devtools) page for
 ### Writing tests
 1. Create a new test file in `frontend/cypress/tests`.
 1. At project root start *Cypress Test Runner* by running `gradlew cypress`.
-1. On the top right hand corner, set `Chrome` as the default browser.
-1. Under **Integration Tests**, click on the newly created test file to run it.
-![Cypress Test Runner](../images/cypress-test-runner.jpg "Cypress Test Runner")
+1. Select **E2E Testing** and choose **Chrome** as the preferred browser.
+ ![Cypress Test E2E](../images/cypress-test-e2e.png)
+ ![Cypress Test Chrome](../images/cypress-test-chrome.png)
```

Comment:
Can you align the indentations of the image links here

Reply:
Sorry, are you referring to the 3rd image? From my side the first 2 images look aligned:

<details>
<summary>Screenshot</summary>

![alignment](https://user-images.githubusercontent.com/45852430/224466013-dfc166c7-924a-4c7b-9408-e9a2174ad91e.png)
</details>

Reply:
Oh I mean the indentation of the text in the file itself

Reply:
Oh got it! Sorry for the misunderstanding - will fix this

Reply:
> Oh I mean the indentation of the text in the file itself. The number of whitespaces is different from that of other image links here

Just noticed that the images in lines 84/86 have 3 whitespaces of indentation, but the previous links in this section had none - is there a specific one I should follow? I couldn't find anything in the style guide relating to this

Reply:
I think you can just remove the preceding whitespaces, in the same way as how the multiline content in a list is indented